src/binding.cpp

```diff
@@ -261,14 +262,13 @@ NAN_METHOD(render_sync) {
 
   sass_free_context_wrapper(ctx_w);
 
-  NanReturnValue(NanNew<Boolean>(result == 0));
+  info.GetReturnValue().Set(Nan::New<v8::Boolean>(result == 0));
```

Comment:
On Sun, 9 Aug 2015, Benjamin Byholm wrote:

> > @@ -261,14 +262,13 @@ NAN_METHOD(render_sync) {
> >  
> >    sass_free_context_wrapper(ctx_w);
> >  
> > -  NanReturnValue(NanNew<Boolean>(result == 0));
> > +  info.GetReturnValue().Set(Nan::New<v8::Boolean>(result == 0));
> 
> can do `info.GetReturnValue().Set(result == 0);` directly

Is it Nan::New never necessary in those cases? (also for strings etc.?)
In that case we have plenty of these, will remove.

Thanks for taking time to review this!